<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParserWindow</class>
 <widget class="QMainWindow" name="ParserWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ParserWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>361</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="mailEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>361</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Download">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Download</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>10</y>
      <width>361</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>URL:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="urlLineEdit">
       <property name="text">
        <string>http://pmg.org.ru/nehe/index.html</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
